### PR TITLE
[zh]Sync content/zh/docs/tasks/traffic-management/egress/egress-tls-origi…

### PR DESCRIPTION
**Pull request by yanrongshi**
Sync upstream for adding `Mutual TLS origination for egress traffic`
```yaml
content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md
```
**en**

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md [hunk 1]
```diff
@@ -178,18 +178,140 @@ test: yes
...
 
-1. 关闭 [sleep]({{< github_tree >}}/samples/sleep) 服务:
+## 出口流量的双向 TLS 发起{#mutual-tls-origination-for-egress-traffic}
+
+本节介绍如何配置 sidecar 为外部服务执行 TLS 发起，这次使用需要双向 TLS 的服务。
+此示例涉及更多，因为它需要以下设置：
+
+1. 生成客户端和服务器证书
+1. 部署支持双向 TLS 协议的外部服务
+1. 将客户端（睡眠 Pod）配置为使用在步骤 1 中创建的凭据
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md:197]:
```suggestion
1. 将客户端（sleep Pod）配置为使用在步骤 1 中创建的凭据
```

Comment (kebe7jun) [reply content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md:197]: 这种不要翻译。

File: content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md [hunk 2]
```diff
@@ -178,18 +178,140 @@ test: yes
...
+
+完成此设置后，您可以将外部流量配置为通过将执行 TLS 发起的边车。
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md:199]: 边车不翻译就一直不翻译。

File: content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md [hunk 3]
```diff
@@ -178,18 +178,140 @@ test: yes
...
+
+### 生成客户端和服务器证书和密钥{#generate-client-and-server-certificates-and-keys}
+
+按照 Egress Gateway TLS 发起任务中的[这些步骤](/zh/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/#generate-client-and-server-certificates-and-keys)进行操作。
+
+### 部署双向 TLS 服务器{#deploy-a-mutual-tls-server}
+
+按照 Egress Gateway TLS 发起任务中的[这些步骤](/zh/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/#deploy-a-mutual-tls-server)进行操作。
+
+### 配置客户端——睡眠 Pod{#configure-the-client-sleep-pod}
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md:209]: sleep

File: content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md [hunk 4]
```diff
@@ -178,18 +178,140 @@ test: yes
...
+    **必须**在部署客户端 Pod 的统一命名空间中创建密钥，本例为 `default`。
+
+1. 创建必需的 `RBAC` 以确保在上述步骤中创建的密钥对客户端 Pod 是可访问的，在本例中是 `sleep`。
+
+    {{< text bash >}}
+    $ kubectl create role client-credential-role --resource=secret --verb=list
+    $ kubectl create rolebinding client-credential-role-binding --role=client-credential-role --serviceaccount=default:sleep
+    {{< /text >}}
+
+### 为边车上的出口流量配置双向 TLS 发起{#configure-mutual-tls-origination-for-egress-traffic-at-sidecar}
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md:227]: 注意边车。

File: content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md [hunk 5]
```diff
@@ -178,18 +178,140 @@ test: yes
...
     $ kubectl delete serviceentry edition-cnn-com
-    $ kubectl delete virtualservice edition-cnn-com
     $ kubectl delete destinationrule edition-cnn-com
     {{< /text >}}
 
-1. 关闭 [sleep]({{< github_tree >}}/samples/sleep) 服务:
+## 出口流量的双向 TLS 发起{#mutual-tls-origination-for-egress-traffic}
+
+本节介绍如何配置 sidecar 为外部服务执行 TLS 发起，这次使用需要双向 TLS 的服务。
+此示例涉及更多，因为它需要以下设置：
```
Comment (windsonsea) [inline content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md:193]:
```suggestion
此示例涉及许多内容，需要先执行以下前置操作：
```

File: content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md (hunk 2 above)
Comment (windsonsea) [inline content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md:199]:
```suggestion
完成上述前置操作后，您可以将外部流量配置为经由该 sidecar，执行 TLS 发起。
```

File: content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md [hunk 6]
```diff
@@ -178,18 +178,140 @@ test: yes
...
+
+1. 生成客户端和服务器证书
+1. 部署支持双向 TLS 协议的外部服务
+1. 将客户端（睡眠 Pod）配置为使用在步骤 1 中创建的凭据
+
+完成此设置后，您可以将外部流量配置为通过将执行 TLS 发起的边车。
+
+### 生成客户端和服务器证书和密钥{#generate-client-and-server-certificates-and-keys}
```
Comment (windsonsea) [inline content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md:201]:
```suggestion
### 生成客户端证书、服务器证书、客户端密钥和服务器密钥{#generate-client-and-server-certificates-and-keys}
```

File: content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md [hunk 7]
```diff
@@ -178,18 +178,140 @@ test: yes
...
+    $ kubectl delete destinationrule originate-mtls-for-nginx
+    {{< /text >}}
+
+1.  删除证书和私钥：
+
+    {{< text bash >}}
+    $ rm example.com.crt example.com.key my-nginx.mesh-external.svc.cluster.local.crt my-nginx.mesh-external.svc.cluster.local.key my-nginx.mesh-external.svc.cluster.local.csr client.example.com.crt client.example.com.csr client.example.com.key
+    {{< /text >}}
+
+1.  删除本示例中使用的生成的配置文件：
```
Comment (windsonsea) [inline content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md:299]:
```suggestion
1.  删除本示例中用过和生成的那些配置文件：
```

**Review by windsonsea — APPROVED** [review]

**Pull request merged**